akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -518,7 +519,7 @@ private[stream] final class GraphInterpreter(
         eventsRemaining -= 1
       }
     } finally {
-      setCurrentInterpreter(previousInterpreter)
+      currentInterpreterHolder(0) = previousInterpreter
```

Comment:
After this change `setCurrentInterpreter`, `currentInterpreterOrNull` and `currentInterpreter` is not used anymore.